### ISSUE TITLE
How to get obj p2d?Using 3d point projection?

### ISSUE THREAD

**Issue by lzfff12** [stevenlsw/Semi-Hand-Object#10]

**Comment by stevenlsw** [comment]
yes, use the 2d point could be obtained by projection of camera matrix

**Comment by lzfff12** [comment]
Thanks

**Comment by lzfff12** [comment]
I have another question. If I want to calculate the pose of the object during training, can I directly use the code during the test? Because the image has been rotated data enhancement, can 2D and 3D correspond?

**Comment by stevenlsw** [comment]
Sure, as long as you figure out the 2D transformation and transform the
projected points to the right location in the image. One thing to notice is
that at inference time, the PNP is not differentiable.

Best,
Shaowei

**Comment by lzfff12** [comment]
Thanks

**Comment by lzfff12** [comment]
Can you provide the code for visualizing the attention map in the paper
